cicd/pr_check_template.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+APP_NAME="CHANGEME"  # name of app-sre "application" folder this component lives in
+COMPONENT_NAME="CHANGEME"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/CHANGEME"  # This is the image location on quay
+
+IQE_PLUGINS="CHANGEME"  # name of the IQE plugin for this APP
+IQE_MARKER_EXPRESSION="smoke"  # This is the value passed to pytest -m
+IQE_FILTER_EXPRESSION=""  # This is the value passed to pytest -k
+
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh -o bootstrap.sh
+
+# The contents of the bootstrap script can be found here:
+# https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd/bootstrap.sh
+# This script automates the install / config of bonfire, the DOCs covering this can be found:
+# https://internal.cloud.redhat.com/docs/devprod/ephemeral/01-onboarding/
+source bootstrap.sh  # checks out bonfire and changes to "cicd" dir...
+
+# The contents of build.sh can be found at:
+# https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd/build.sh
+# This script is used to build the image that is used in the PR Check
+source build.sh
+
+# Your APP's unit tests should be run in the unit_test.sh script.  Two different
+# examples of unit_test.sh are provided in:
+# https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd/unit_test_example_django.sh
```

Comment:
Technically this could apply to more than just django

Maybe `unit_test_example_ephemeral_db.sh` and `unit_test_example.sh` ?